editor/src/com/talosvfx/talos/editor/addons/scene/logic/components/SpineRendererComponent.java

```diff
@@ -133,6 +134,11 @@ private void createSkeletonFromGameAsset () {
     @Override
     public void setGameAsset (GameAsset<SkeletonData> gameAsset) {
         this.gameAsset = gameAsset;
+
+        if(defaultGameAsset == null && !gameAsset.nameIdentifier.equals("broken")){
```

Comment:
What is the purpose of this?

Reply:
when we import sprite at first we add broken then valid one

Reply:
You are storing default for reset functionality?